### ISSUE TITLE
Using background image(No object image) for validation data while Azure AutoML Object Dectection Train

### ISSUE THREAD

**Issue by Sup90** [Azure/azureml-examples#2233]
### Describe your suggestion

Hi
I want to check azure automl object dectection model's False Positve performance whie model training. 
so i added background image(No object image) to validation data
![image](https://user-images.githubusercontent.com/30144765/234763572-92fe0d51-d1d0-411e-b350-f8e239e72ef3.png)
in confusion metric of model's metric page, but there is no predict result for background image.
(sum of confusion metric value is same with number of object image(except no object image)
Please let me know how to add background image(No object image) for perfomance metric.

**Comment by rdondera-microsoft** [comment]
Hi Sup90, the confusion matrix for object detection shows
a. how many ground truth objects were correctly detected in terms of both localization and class (elements on the diagonal)
b. how many ground truth objects were correctly detected in terms of localization but not class (elements off diagonal, except the last column)
c. how many ground truth objects were not detected (last column)
The confusion matrix rows correspond to the ground truth classses. Adding background images does not change the confusion matrix since the set of ground truth objects stays the same.

We do not expose the number of false positives directly, but instead report precision at the highest recall value (the "precision" metric) and we have plans to report precision at multiple score threshold values in the next release. Let us know if this addresses your needs.

**Comment by Sup90** [comment]
Hi @vadthyavath 
Thank you for your kind and prompt response. I have one additional question. I am creating an Object Detection model using Faster R-CNN. I wonder if adding background images to the training data without labeling information would actually help improve the model's False Positive (FP) rate.

I have empirically observed performance improvements when adding some of these background images during changes in the training model version, but I don't have a way to validate this. I want to confirm whether this approach is technically and theoretically reasonable within Azure AutoML.

Thank you.

**Comment by rdondera-microsoft** [comment]
Hi @Sup90, sorry for the delayed response. Yes, adding images without objects should help improve the FPR, but the exact amount depends on problem specifics. We expect that background images would help defect detection tasks, since they allow training to use more relevant boxes of the "non-defective" kind. Also, note that the idea to use background images is not model specific, see this comment for Yolo: https://github.com/ultralytics/yolov5/issues/2844#issuecomment-851338384 .
